app/src/main/java/com/oracle/javafx/scenebuilder/app/preferences/PreferencesImporter.java

```diff
@@ -180,14 +176,24 @@ public void tryImportingPreviousVersionSettings() {
      * @return true when previous version settings have been found and user has not yet decided 
      */
     public boolean askForImportIfOlderSettingsExist() {
-        return this.optionalSourceNode.map(this::previousVersionPrefsFound).isPresent() && askForImport();
+        boolean previousVersionFound = this.optionalSourceNode.isPresent();
+        logger.log(Level.FINE, "older preferences detected: {0}", previousVersionFound);
+        if (previousVersionFound) {
+            VersionedPreferences prefs = this.optionalSourceNode.get();
+            logger.log(Level.FINE, "Version: {0}", prefs.version());
+            logger.log(Level.FINE, "Node   : {0}", prefs.node());
+        }
+        boolean forcedImport = importForced();
+        if (forcedImport) {
+            logger.log(Level.FINE, "detected -DforceImport=true");
+        }
+        return previousVersionFound && (forcedImport || askForImport());
     }
 
-    private VersionedPreferences previousVersionPrefsFound(VersionedPreferences versionedpreferences) {
-        logger.log(Level.FINE, "{0}", versionedpreferences);
-        return versionedpreferences;
+    private boolean importForced() {
```

Comment:
IMO, a better name for this method would be `isImportForced`

Reply:
Before renaming it, as I've added this mainly for debugging.
is this option something desirable to keep in the code?

Reply:
To me, it makes sense assuming functionality of this PR is accepted.

Reply:
Okay, good to know. Actually, is it possible to pass properties to the execution when staring Scene Builder using Maven? I tried to run `mvn javafx:run -f app -DforceImport=true` and unfortunately this does not work.

Reply:
If I'm not mistaken, you have to pass a property for javafx maven plugin. 
Since command line args for the plugin are under `<commandlineArgs>` tag, your argument should look somewhat like: `-DcommandlineArgs="-DforceImport=true"`. 
I haven't tested this, but this is how Maven passed arguments to plugins.

Reply:
Thank you - I tried this but unfortunately this did not work. However, running the Jar manually or running from within IDE works.